glib-sys/src/manual.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2013-2018, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+#[allow(unused_imports)]
+use libc::{c_int, c_ushort};
+
+#[cfg(all(windows,target_arch="x86_64"))]
+pub type GPid = i64; //Actually HANDLE aka void*
```

Comment:
Shouldn't this be a `*mut c_void` then?

Reply:
It depend of https://github.com/gtk-rs/sys/issues/82 discussion